tests/test_create_wif_attestation.cpp

```diff
@@ -78,22 +103,31 @@ long run_request_curl(
   return response_code;
 }
 
-void test_aws_attestation(void**)
-{
-  char* gh_actions = std::getenv("GITHUB_ACTIONS");
+const std::string AWS_TEST_REGION = "us-east-1";
+const std::string AWS_TEST_ARN = "arn:aws:iam::123456789012:user/abcd1234";
```

Comment:
nit: use the ARN format that we'll get in this case
```suggestion
const std::string AWS_TEST_ARN = "arn:aws:sts::123456789012:assumed-role/my-role/session-abc";
```